packages/scandipwa/src/component/AddToCart/AddToCart.container.js

```diff
@@ -137,6 +139,21 @@ export class AddToCartContainer extends PureComponent {
         return true;
     }
 
+    validateDownloadableProduct() {
+        const {
+            product: { links_purchased_separately },
+            productOptionsData: { downloadableLinks = [] },
+            showNotification
+        } = this.props;
+
+        if (links_purchased_separately && downloadableLinks.length === 0) {
+            showNotification('info', __('Please, select product links!'));
+            return false;
```

Comment:
I fixed it throughout the project for now (using regexp).

@IvansZuks but why don't we enforce this rule as a ESLint rule? It should make everyone's life easier.

Reply:
There are a lot of eslint rules, that are present in the scandipwa scripts, but are disabled for the scandipwa core.
Likely, because there are a lot of legacy code and we should refactor all project to match these rules.
I have a dream, that once we will enable them all.